Dockerfile

```diff
@@ -28,6 +28,19 @@ RUN yarn install --frozen-lockfile && \
     yarn sass scss/_main.scss /app/marsha/static/css/main.css --style=compressed --load-path=node_modules  && \
     yarn build --mode=production --output-path /app/marsha/static/js/build/
 
+# ---- mails ----
+FROM node:14 as mail-builder
+RUN mkdir -p /app/backend/marsha/core/templates/core/mail/html/ && \
+    mkdir -p /app/backend/marsha/core/templates/core/mail/text/ && \
+    mkdir -p /app/mail
+
+COPY ./src/mail /app/mail
+
+WORKDIR /app/mail
+
+RUN yarn install --frozen-lockfile && \
+    yarn build-mails
+
```

Comment:
I'm not sure this is necessary. If the templates are updated by the developper, we can do the same as with translations ie commit the html result in the project?
If you prefer to keep this, then we should not commit the html version and gitignore everything in this subdirectory of `templates`? But I think it may be dangerous and it's better that the developper manually generates the html files each time s.he changes something in the mjml?

Reply:
At the moment I stopped committed the HTML and text files, I don't know which way is better, wdyt @lunika , do we commit the generated files and stop generating them in the docker file or do we ignore them in the gitignore and leave it this way ?

Reply:
Both strategy are valid but most of the time we don't commit generated files in the project. This is the case for the javascript generated by webpack, the translations generated in the build, the css generated by the sass compiler, and so on.
We need to generate them in the development environment to test them but commiting them can also be error prone. You can forget to commit the generated modification. We should in that case add a new check in CI to be sure you don't forget to do it.

Reply:
True that :thinking: 
And one CI check for the i8n also :laughing: 
So maybe the Dockerfile strategy is better as long as we also have a way to work on the mjml in realtime.

Reply:
The `.gitignore` file must be updated in that case.